### DISCUSSION TITLE
Bug \ display artifact 3d model

### DISCUSSION THREAD

**Discussion by IlyaNavodkin** [helix-editor/helix#4623] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88714829/200197828-0d9cde8e-ba6f-4390-8d91-ca213c436da0.png)
![image](https://user-images.githubusercontent.com/88714829/200197969-c8128508-8ad2-4887-9795-f8cb31604dd3.png)

Hello!

The display of elements is incorrect, as if it is multiplying (no overlays).

What can be wrong?

**Comment by the-mikedavis** [comment, marked as answer]
This is not the correct repository: the Helix in this repository is a text editor.
